doc/instantsend.md

```diff
@@ -27,45 +27,11 @@ When a wallet InstantSend transaction is successfully locked a shell command pro
 
 Details pertaining to an observed "Transaction Lock" can also be retrieved through RPC, it’s important however to understand the underlying mechanism.
 
-By default, the Dash Core daemon will launch using the following constant:
-
-```
-static const int DEFAULT_INSTANTSEND_DEPTH = 5;
-```
-
-This value can be overridden by passing the following argument to the Dash Core daemon:
-
-```
--instantsenddepth=<n>
-```
-
-The key thing to understand is that this value indicates the number of "confirmations" a successful Transaction Lock represents. When Wallet RPC commands which support `minconf` and `addlockconf` parameters (such as `listreceivedbyaddress`) are performed and `addlockconf` is `true`, then `instantsenddepth` attribute is taken into account when returning information about the transaction. In this case the value in `confirmations` field you see through RPC is showing the number of `"Blockchain Confirmations" + "InstantSend Depth"` (assuming the funds were sent via InstantSend).
-
-There is also a field named `instantlock` (that is present in commands such as `listsinceblock`). The value in this field indicates whether a given transaction is locked via InstantSend.
+There is a field named `instantlock` (that is present in commands such as `listsinceblock`). The value in this field indicates whether a given transaction is locked via InstantSend.
```

Comment:
Looks pretty good. But there is no `underlying mechanism` anymore and `Examples` sub-section doesn't really give any additional valuable info now, so we probably can write the whole RPC section like this:
> Details pertaining to an observed "Transaction Lock" can also be retrieved through RPC. There is a boolean field named `instantlock` which indicates whether a given transaction is locked via InstantSend. This field is present in the output of some wallet RPC commands e.g. `listsinceblock`, `gettransaction` etc. as well as in the output of some mempool RPC commands e.g. `getmempoolentry` and a couple of others like `getrawmempool` (for `verbose=true` only).

Thoughts? @codablock @nmarley

Reply:
I agree, this paragraph seems more accurate (removes "underlying mechanism") and descriptive (adds info on boolean field and which RPC commands have it).